src/contracts/FieldType/DefaultDataFieldStorage.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\FieldType;
```

Comment:
This namespace is not correct. `./src/contracts` is mapped to `Ibexa\Contracts\Core\`.
Moreover I would move it one level down, to:
```suggestion
namespace Ibexa\Contracts\Core\FieldType\Storage;
```